concepticondata/concepticon.tsv

```diff
@@ -2785,3 +2785,4 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 2784	CRUCIAN CARP	Animals	A medium-sized member of the common carp family.	Person/Thing	
 2785	BIVALVIA (OYSTERS OR MUSSELS)	Animals	A class of marine and freshwater molluscs that have laterally compressed bodies enclosed by a shell consisting of two hinged parts.	Person/Thing	
 2786	CLOG (SOMETHING)	Basic actions and technology	To prevent something from getting through.	Action/Process	
+2787	DRINK OR EAT	Food and drink	To consume food or drink.	Action/Process	
```

Comment:
finally we have this concept, but I'd recommend to rename into: CONSUME (DRINK OR EAT), as this is a gloss you sometimes find in the literature (the *consume* part), and if possble, we try to find a gloss which generalizes (such as MUROID (MOUSE OR RAT), etc.).